common/app/views/support/HtmlCleaner.scala

```diff
@@ -500,6 +500,16 @@ case class DropCaps(isFeature: Boolean) extends HtmlCleaner {
         case _ =>
       }
     }
+
+    document.getElementsByTag("h2").foreach{ h2 =>
+        if (h2.text() == "* * *") {
```

Comment:
Maybe add a comment explaining the intent, the context is a little bit implicit.